### ISSUE TITLE
Suggest extraneous function wrap in an impossible situation

### ISSUE THREAD

**Issue by gunar** [jfmengels/eslint-plugin-lodash-fp#39]
I can't just rely no currying because I need the passed argument to be reflected in two places.

```js
const pickId = mapValues(v => getOr(v, '_id', v)) // <- This generates a warning.
```

**Comment by jfmengels** [comment]
Hi @gunar!

Which version of this plugin are you using? Because I have fixed this behavior in the latest 2.0.1 version. Try updating your plugin, and let me know if that still doesn't work :)

**Comment by gunar** [comment]
It does indeed. Thank you!
